ui/src/main/scripts/plugins/btm/ts/btm.ts

```diff
@@ -163,6 +169,24 @@ module BTM {
       $scope.faultChartHasData = txnFaultData.length > 0;
     };
 
+    $scope.addPropertyToFilter = function(pName, pValue, operator) {
```

Comment:
Yes, that (and a couple more methods below) are methods which are to be invoked by the sidebar. Initial goal was that each usage could have it's own implementation but I think it's not happening, and probably it can be _moved up_ to the directive. Will investigate that later.